### ISSUE TITLE
Cannot render fonts when using build

### ISSUE THREAD

**Issue by peet-j** [podigee/podigee-podcast-player#54]
I've been building the project then pushing the created artifacts (everything inside build/) to S3.
When accessing /podigee-podcast-file.html and using the configuration query parameter (or for that matter using the script tag embed), the player renders and functions correctly.
However the fonts are not loading correctly. See screenshot:

![screen shot 2018-09-11 at 14 09 52](https://user-images.githubusercontent.com/33906462/45362355-84287580-b5cc-11e8-8ef6-04908e324300.png)

**Comment by benzimmer** [comment]
@peet-j Can you post a snippet how exactly you configure the player?

**Comment by peet-j** [comment]
Thanks for the quick response @benzimmer.

So, interestingly if I point to your github hosted pages like this:

http://podigee.github.io/podigee-podcast-player/build/podigee-podcast-player.html?configuration=http://podigee.github.io/podigee-podcast-player/example/config.json

then the fonts are loaded correctly. However using the below, the icons are not rendered:

https://s3-eu-west-1.amazonaws.com/james-embed-player/build/podigee-podcast-player.html?configuration=https://podigee.github.io/podigee-podcast-player/example/config.json

BUT, the following page renders fine: https://s3-eu-west-1.amazonaws.com/james-embed-player/index.html



Is there some difference to the github pages code and that in the master branch?

**Comment by benzimmer** [comment]
It might have something to do with the content type the font is served with from S3. In your case it's `application/x-font-ttf`. With the Github hosted page it's `font/ttf`. That would be the only difference I notice. I think you should be able to change the type in S3 though. Maybe that's worth a shot.

**Comment by peet-j** [comment]
Yes I did wonder about that too. I've given that a try but it's still not loading. 
Have also tried GCP storage and am having the same behaviour.
What's strange is that the index.html loads the same font file fine.

**Comment by dirkholzapfel** [comment]
@peet-j we ran into the same problem. we could solve the issue the following way:

The file `podigee-podcast-player.html` has to be served with Content-Type `text/html; charset=utf-8`, NOT with `text/html`

See https://github.com/podigee/podigee-podcast-player/pull/56

**Comment by peet-j** [comment]
@dirkholzapfel awesome, thanks for the tip
